pkg/proxy/metrics/metrics.go

```diff
@@ -126,6 +126,15 @@ var (
 			Help:      "Cumulative proxy iptables restore failures",
 		},
 	)
+
+	// NodeChangesTotal is the number of node changes that the proxy has seen.
+	NodeChangesTotal = prometheus.NewCounter(
```

Comment:
Maybe I missed it, but where is this counter being incremented?

Reply:
Also I think you need to register this as part of `RegisterMetrics`?

Reply:
one more nit: can we group this next to `ServiceChangesTotal` or `EndpointsChangesTotal`?

Reply:
Oops, nice catch.